_Group Projects/Group_5/README.Rmd

```diff
@@ -0,0 +1,17 @@
+---
+title: "README"
+output: md_document
+date: "`r Sys.Date()`"
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
```

Comment:
If you wanted to simplify this process/skip having to use R, you could use a .md file type and then you could remove all this code. I'm also a bit skeptical that a .Rmd file like this will render nicely on Github, but I could be wrong about that. For example, see example of source code and rendered .md file on Github: [source code](https://raw.githubusercontent.com/worldbank/dime-standards/df0a618332d3f24b8d526a7b8082ccc28b622e31/dime-coding-standards/checklists/analysis-code-review.md), [what it looks like on Github](https://github.com/worldbank/dime-standards/blob/master/dime-coding-standards/checklists/analysis-code-review.md)

Reply:
Changed format to .md, as requested.